modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/AlertTag.java

```diff
@@ -125,27 +79,10 @@ public void setMessage(String message) {
 		_message = message;
 	}
 
-	/**
-	 * @deprecated As of Athanasius (7.3.x), replaced by {@link
-	 *             #setDisplayType(String)}
-	 */
-	@Deprecated
-	public void setStyle(String style) {
-		setDisplayType(style);
-	}
-
 	public void setTitle(String title) {
 		_title = title;
 	}
 
-	public void setType(String type) {
-		_type = type;
-	}
-
-	/**
-	 * @deprecated As of Athanasius (7.3.x), with no direct replacement
-	 */
-	@Deprecated
```

Comment:
Remember to comment that this is because it's used by `StripeTag` so we un-deprecate